tests/test_ensemble.py

```diff
@@ -31,6 +35,76 @@ def test_policy_loading_simple(tmpdir):
     assert original_policy_ensemble.policies == loaded_policy_ensemble.policies
 
 
+class Priority_1_Policy(Policy):
+    def __init__(self,
+                 priority: int = 1,
+                 ) -> None:
+        super(Priority_1_Policy, self).__init__(priority=priority)
+
+    @classmethod
+    def load(cls, path):
+        pass
+
+    def persist(self, path):
+        pass
+
+    def train(self, training_trackers, domain, **kwargs):
+        pass
+
+    def predict_action_probabilities(self, tracker, domain):
+        result = [0.0] * domain.num_actions
+        result[0] = 1.0
+        return result
+
+
+class Priority_2_Policy(Policy):
```

Comment:
I think this should not be duplicated, rather the above policy shoud be called `ConstantPolicy` or something like that and get the index to predict as an init param.